types/queries/insert_chapter_mark_type_genre.sql

```diff
@@ -0,0 +1,17 @@
+insert into collection_chapter (
+  title,
+  document_id,
+  wordpress_id,
+  index_in_parent,
+  chapter_path,
+  section)
+values (
+  $1,
+  ( select id
+    from document
+    where short_name = $2),
+  $3,
+  $4,
+  $5,
+  'Body'
```

Comment:
oh yeah, regarding your point about those two files above, I can just delete them since they're irrelevant now and were from an earlier iteration of the code-- I think that would probably be best?

Reply:
Yeah, I think so. And since they're not very complex, we could easily create something similar if we ever needed in the future.

Reply:
OK, should be all set thanks!